src/sbuf.c

```diff
@@ -1235,6 +1236,44 @@ static bool setup_tls(struct tls_config *conf, const char *pfx, int sslmode,
 	return true;
 }
 
+static bool tls_config_changed(struct tls_config *new_server_connect_conf)
+{
+  return (strings_equal(new_server_connect_conf[0].ca_file, server_connect_conf->ca_file) &&
```

Comment:
I wish C had reflection :) Alas. In any case, there are two members/structs I didn't check here worth noting. [`tls_error`](https://github.com/libusual/libusual/blob/master/usual/tls/tls_internal.h#L50) which seems to be for internal handling of TLS setup errors. Also, [`tls_keypair`](tls_keypair). keypair appears to be a list of keys. 

Maybe it should be checked? I would need to iterate through both the new and the old which is fine.